From e879b73a732e0f3d88624994bc69bcb9cd56b284 Mon Sep 17 00:00:00 2001
From: Salvatore Bonaccorso <carnil@debian.org>
Date: Tue, 29 May 2012 19:51:46 +0200
Subject: [PATCH] bts: When searching for usertags use tag= in the url

When searching for usertags the url does not contain usertag= but tag=
followed by the options (containing users=). See [1].

 [1]: https://lists.debian.org/debian-devel-announce/2005/09/msg00002.html

Closes: #675071
---
 debian/changelog |    4 ++++
 scripts/bts.pl   |    2 +-
 2 files changed, 5 insertions(+), 1 deletion(-)

diff --git a/debian/changelog b/debian/changelog
index 4772cb2..a303fc7 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -40,6 +40,10 @@ devscripts (2.11.8) UNRELEASED; urgency=low
   * devscripts.Logger Don't substitute arguments into logged strings unless
     they were provided. (LP: #968129)
 
+  [ Salvatore Bonaccorso ]
+  * bts: When searching for usertags use tag= in the url (followed by
+    the options containing users=). (Closes: #675071).
+
  -- Benjamin Drung <bdrung@debian.org>  Mon, 28 May 2012 17:14:12 +0200
 
 devscripts (2.11.7) unstable; urgency=low
diff --git a/scripts/bts.pl b/scripts/bts.pl
index 36f0d2d..c326898 100755
--- a/scripts/bts.pl
+++ b/scripts/bts.pl
@@ -3399,7 +3399,7 @@ sub thing_to_url {
 	($thingurl = $thing) =~ s/^tag:/tag=/;
 	$thingurl = $btscgipkgurl.'?'.$thingurl;
     } elsif ($thing =~ /^usertag:/) {
-	($thingurl = $thing) =~ s/^usertag:/usertag=/;
+	($thingurl = $thing) =~ s/^usertag:/tag=/;
 	$thingurl = $btscgipkgurl.'?'.$thingurl;
     } elsif ($thing =~ m%^release-critical(\.html|/(index\.html)?)?$%) {
 	$thingurl = $btsurl . 'release-critical/index.html';
-- 
1.7.10

